### PR TITLE
添加scan/login事件

### PR DESCRIPTION
**Pull request by lyleshaw**
（还未整理，代码有些乱）

### CODE REVIEW

File: src/wechaty_puppet_itchat/puppet.py [hunk 1]
```diff
@@ -166,34 +167,40 @@ def __init__(self, options: PuppetOptions, name: str = 'puppet_itchat'):
         Raises:
             WechatyPuppetConfigurationError: raise Error when configuraiton occur error
         """
-        super().__init__(options, name)
+        super().__init__(name)
 
         self.channel: Optional[Channel] = None
-        self._puppet_stub: Optional[PuppetStub] = None
```
Comment (lyleshaw) [inline src/wechaty_puppet_itchat/puppet.py:172]: 这里的修改是为了让wechaty能跑通（由于puppet-itchat没有token）

**Review by wj-Mcat — APPROVED** [review]
LGTM. Let's merge it as a milestone and we should try to fix the next problem.

File: src/wechaty_puppet_itchat/puppet.py (hunk 1 above)
Comment (wj-Mcat) [reply src/wechaty_puppet_itchat/puppet.py:172]: Yes, you are right. we should not use token anymore.

**Comment by wj-Mcat** [comment]
you can merge it first, and create an issue to describe the next scan problem,

**Comment by lyleshaw** [comment]
> you can merge it first, and create an issue to describe the next scan problem. So we can try to create pr to fix the created issue.

Done

**Pull request merged**